package.json

```diff
@@ -226,8 +218,10 @@
       "built": true
     }
   },
+  "pnpm": {
+    "onlyBuiltDependencies": []
+  },
   "resolutions": {
-    "@bazel/concatjs@npm:5.8.1": "patch:@bazel/concatjs@npm%3A5.8.1#~/.yarn/patches/@bazel-concatjs-npm-5.8.1-1bf81df846.patch",
-    "@microsoft/api-extractor/typescript": "5.6.3"
+    "typescript": "5.6.3"
```

Comment:
Question, is this still needed?

Reply:
Yeah, we still need it. Updated to the latest version